src/main/java/com/exactpro/th2/fix/client/fixBean/BaseFixBean.java

```diff
@@ -0,0 +1,161 @@
+package com.exactpro.th2.fix.client.fixBean;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+import java.util.regex.Pattern;
+
+import static com.exactpro.th2.fix.client.util.FixBeanUtil.addToConfig;
+
+public class BaseFixBean {
+
+    private static final Pattern YES_OR_NO = Pattern.compile("[YN]?");
```

Comment:
There's an easier way of checking if string equals to one symbol or another.
And `[YN]?` also matches empty string